org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,89 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: raceintospace
+#rename-icon: raceintospace
+rename-desktop-file: raceintospace.desktop
```

Comment:
These renames should ideally happen upstream.

Reply:
Flatpak is not yet primary source of installation. And  it was historically in current paths, I would not change it right away.

Reply:
IIRC both the desktop file & appstream specs suggests using a reverse domain name for the file names see https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#file-naming & https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic.
It's just Flatpak enforces that every exported file should be prefixed by the app-id.

Reply:
At the same time, application had raceintospace.desktop id in ancient history. Icon change would not affect installations into /usr/local, but desktop name change would. I am not sure we should move it on upstream side now. I doubt there would be any real benefit in that. I were unable to display my meta file in gnome-software until now. Guess there is something wrong with that, which I cannot find.

Reply:
Because any store out there doesn't use the metainfo provided by the application, they are combined in an appstream collection file that's defined per repository (otherwise you will need to download the app before being able to know more details about that app). If you're running a test build or a local build, you won't be able to see any information about the locally built package.

Reply:
I pushed this application to official [Fedora package](https://src.fedoraproject.org/rpms/raceintospace), but even then I were unable to display my metadata. I am not sure what is correct process to add metadata to the distribution. I thought it is just read by gnome-software from /usr/share/metainfo. But what you say actually makes sense, it has to be somewhere to offer nice data before installation.